src/app/dashboard/create_budget_form.tsx

```diff
@@ -10,10 +10,15 @@ export function NewBudgetForm({user_id}: {user_id: string}) {
 
   const [name, setName] = useState('Dance Party');
   const [description, setDescription] = useState('Lots of Dancing and Fun');
+  const [location, setLocation] = useState('HSSC N114');
+  const [date, setDate] = useState('2024-MM-DD');
+  const [type, setType] = useState('SEPC');
```

Comment:
the default value for `date` should be today's date (since an event can be planned for the past) and for the rest. that is, it should be `useState("")` all the properties except date and `useState(new Date().toISOString()))` for the date (the function `new Date() ` creates a new `Date` object based on the current time and `toISOString()` converts it into a well-formatted string we will store in our database